python/runtime/cudaq/algorithms/py_observe.cpp

```diff
@@ -185,75 +196,110 @@ async_observe_result pyObserveAsync(kernel_builder<> &kernel,
 void bindObserve(py::module &mod) {
   auto parallelSubmodule = mod.def_submodule("par");
 
-  py::class_<cudaq::par::mpi>(parallelSubmodule, "mpi");
-  py::class_<cudaq::par::thread>(parallelSubmodule, "thread");
+  py::class_<cudaq::par::mpi>(
```

Comment:
We should connect these types, and any other new API code, to the [python api file](https://github.com/NVIDIA/cuda-quantum/blob/main/docs/sphinx/api/languages/python_api.rst)